### PR TITLE
feat(challenges): Add default project image when none exists

### LINKED ISSUE #16426: Add default image to Projects
Same project challenges do not provide an image. We will need a default image to render here instead of nothing.

see: #16410

### PR DESCRIPTION
**Pull request by jameskaupert**
BREAKING CHANGE: none

Closes #16426

#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [x] Tested changes locally. Tested in both Chrome and Firefox.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16426

#### Description
Add a default image from Imgur of FCC fire on `#006400` dark green background when project submission page does not have an image associated with it.

### CODE REVIEW

File: common/app/routes/Challenges/views/project/Project.jsx [hunk 1]
```diff
@@ -45,7 +45,8 @@ export class Project extends PureComponent {
       isCompleted,
       description
     } = this.props;
-    const imageURL = '//i.imgur.com/' + image + '.png';
+    const imgurId = image ? image : 'ovKSXMs';
+    const imageURL = '//i.imgur.com/' + imgurId + '.png';
```
Comment (BerkeleyTrue) [inline common/app/routes/Challenges/views/project/Project.jsx]: We can make this simpler by using default expressions. Then the other changes become unnecessary.
```js
    const {
      id,
      title,
      image = 'KDJFLDK',
      isCompleted,
      description
    } = this.props;
```

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]

File: common/app/routes/Challenges/views/project/Project.jsx (hunk 1 above)
Comment (jameskaupert) [reply common/app/routes/Challenges/views/project/Project.jsx]: Will fix. Thanks!

Comment (jameskaupert) [reply common/app/routes/Challenges/views/project/Project.jsx]: Updated to add the default image as outlined in the feedback.

Comment (jameskaupert) [reply common/app/routes/Challenges/views/project/Project.jsx]: I went through the process outlined on the Contribute page for amending the PR and force pushed, but I am not seeing my latest updates show up on the PR.  I may have to review this again after work today.

Comment (BerkeleyTrue) [reply common/app/routes/Challenges/views/project/Project.jsx]: did you force push up?

Comment (raisedadead) [reply common/app/routes/Challenges/views/project/Project.jsx]: You may have missed adding the file too? Happens all the time 

```
git add <filename>
git commit --amend --no-edit
```

The force push to the branch this is already on.

**Comment by jameskaupert** [comment]
Should be good to go now.  Thanks @raisedadead for the reminder to `git add`!

File: common/app/routes/Challenges/views/project/Project.jsx (hunk 1 above)
Comment (jameskaupert) [reply common/app/routes/Challenges/views/project/Project.jsx]: I believe I've done this now, but the build failed.  Is there an easy way to tell where to start troubleshooting the failed build?  I'm not really sure where to go next from here.

**Comment by BerkeleyTrue** [comment]
@jameskaupert It may be an issue with travis and our build log. Does `npm run test` complete successfully on your end?

**Comment by jameskaupert** [comment]
Yes, they run now and ran before I pushed as well.

**Comment by raisedadead** [comment]
Build fixed.

**Pull request merged**